Index: sample.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- sample.py	(revision 3ebea9f7d0c53bb7b201a727112b4d8f9d6dbd4e)
+++ sample.py	(date 1588288601870)
@@ -1,4 +1,4 @@
-######!/usr/bin/env python3.7
+#!/usr/bin/env python3.7
 first_name = "rajesh"
 last_name = "raman"
 age = int(55)
